src/main/java/thePackmaster/cards/pinnaclepack/FriendshipCroquettesSpecialColourless.java

```diff
@@ -0,0 +1,39 @@
+package thePackmaster.cards.pinnaclepack;
+
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class FriendshipCroquettesSpecialColourless extends AbstractPinnacleCard {
+
+    public final static String ID = makeID("FriendshipCroquettesSpecialColourless");
+    private static final int MAGIC = 2;
+    private static final int UPGRADE_MAGIC = 2;
+    private static final int BLOCK = 6;
+    private static final int UPGRADE_BLOCK = 6;
+
+
+    public FriendshipCroquettesSpecialColourless() {
```

Comment:
*Friendship Croquette*
This is too complex for a token card and too different from the other Croquette cards. It should be simplified. The most straightforward way to do this is by removing the block increase on retain (which is something none of the other Croquettes have).

Reply:
No, i agreed with GK the croquettes were fine in advance.